modules/managing-memcached-operator-using-olm.adoc

```diff
@@ -28,14 +28,20 @@ Preview OLM enabled
 +
 An Operator manifest describes how to display, create, and manage the
 application, in this case Memcached, as a whole. It is defined by a
-`CustomServiceVersion` (CSV) object and is required for the OLM to function.
+`ClusterServiceVersion` (CSV) object and is required for the OLM to function.
++
+You can use the following command to generate CSV manifests:
++
+----
+$ operator-sdk olm-catalog gen-csv --csv-version 0.0.1
+----
++
+NOTE: This command is run from the `memcached-operator/` directory that was created when you built the Memcached Operator.
```

Comment:
Nit: I think the root directory of the project might be good to add.

Reply:
@LiliC Are you talking about `$GOPATH/src/github.com/example-inc/memcached-operator/` like was used in the above section [1]?

[1] http://file.rdu.redhat.com/~ahoffer/2019/OSDOCS-391/applications/operator_sdk/osdk-getting-started.html#building-memcached-operator-using-osdk-osdk-getting-started

Due to time constraints, I'm going to merge this PR and can follow up in a separate PR if needed.